src/features/search/components/SearchResults.tsx

```diff
@@ -93,6 +95,8 @@ export const SearchResults: React.FC = () => {
   } = useModal(false)
   const { position, showGeolocPermissionModal } = useGeolocation()
   const hasPosition = useHasPosition()
+  const initialLocationRef = useRef<LocationType | undefined>(params?.locationFilter?.locationType)
```

Comment:
Laisse-le deviner comme un grand :)

```suggestion
  const initialLocationRef = useRef(params?.locationFilter?.locationType)
```

Reply:
C'est pas mieux de forcer son typage ?